src/util/psbt/map/output.rs

```diff
@@ -99,6 +99,12 @@ impl core::fmt::Display for IncompleteTapTree {
     }
 }
 
+impl From<IncompleteTapTree> for crate::util::psbt::Error {
+    fn from(i: IncompleteTapTree) -> Self {
+        crate::util::psbt::Error::IncompleteTapTree(i)
```

Comment:
Please use more readable names.

```suggestion
    fn from(error: IncompleteTapTree) -> Self {
        crate::util::psbt::Error::IncompleteTapTree(error)
```